src/site/_includes/components/component-preview/macro.njk

```diff
@@ -3,7 +3,13 @@
 {%- macro guidanceComponentPreview(params) -%}
 
 {%- set exampleTemplatePath = "moduk/components/" + params.componentName + "/__examples__/" + params.exampleName + ".njk" -%}
-{%- set idPrefix = params.componentName + "--" + params.exampleName -%}
+{%- if params.idSuffix -%}
+  {%- set idSuffix = "--" + params.idSuffix -%}
+{%- else -%}
+  {%- set idSuffix = "" -%}
+{%- endif -%}
+
+{%- set idPrefix = (params.idPrefix or '') + params.componentName + "--" + params.exampleName + idSuffix -%}
```

Comment:
Oh sorry didn't update this properly:

```suggestion
{%- set idPrefix = params.componentName + "--" + params.exampleName + idSuffix -%}
```

and then meant change that repeated example in `src/site/components/character-count.md-njk` to pass idSuffix.